.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -205,7 +205,7 @@ stages:
             echo '##vso[task.setvariable variable=IMG_OFFER]UbuntuServer'
             echo '##vso[task.setvariable variable=IMG_SKU]18.04-LTS'
             echo '##vso[task.setvariable variable=HYPERV_GENERATION]V1'
-            echo '##vso[task.setvariable variable=AZURE_VM_SIZE]Standard_NC6s_v3'
+            echo '##vso[task.setvariable variable=AZURE_VM_SIZE]Standard_NC16as_T4_v3'
```

Comment:
are we sure we have quota for all these?

Reply:
Yes, we received everything I requested to facilitate this. 

Planning factor was 4x Test All VHD Pipelines, 5x PR Gates, and 1x Weekly Build all running concurrently.